pkg/client/typed/discovery/discovery_client.go

```diff
@@ -209,75 +211,93 @@ func IsGroupDiscoveryFailedError(err error) bool {
 
 // serverPreferredResources returns the supported resources with the version preferred by the
 // server. If namespaced is true, only namespaced resources will be returned.
-func (d *DiscoveryClient) serverPreferredResources(namespaced bool) ([]schema.GroupVersionResource, error) {
+func (d *DiscoveryClient) serverPreferredResources(namespaced bool) ([]*unversioned.APIResourceList, error) {
```

Comment:
actually, why are you even touching this?  Other than this looking insane, I don't see why your pull would change it.